app/views/atomic_lti/shared/init.html.erb

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script type="text/javascript">
```

Comment:
Add some inline javascript to display an inline error message below in case the browser dies on the javascript.